draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -857,8 +886,62 @@ are not blinded, but contain disclosable claims. This limitation
 needs to be taken into account by Issuers when creating the structure of
 the SD-JWT.
 
+## Issuer Signature Key Distribution and Rotation {#issuer_signature_key_distribution}
+
+This specification does not define how signature verification keys of
+Issuers are distributed to Verifiers. However, it is RECOMMENDED that
+Issuers publish their keys in a way that allows for efficient and secure
+key rotation and revocation, for example, by publishing keys at a
+predefined location using the JSON Web Key Set (JWKS) format [@RFC7517].
+Verifiers need to ensure that they are not using expired or revoked keys
+for signature verification as reasonable and appropriate for the given
+key distribution method.
+
 # Privacy Considerations {#privacy_considerations}
 
+## Storage of Signed User Data
+
+Wherever End-User data is stored, it represents a potentially
+interesting target for an attacker. This target can be of particularly
+high value when the data is signed by a trusted authority like an
+official national identity service. For example, in OpenID Connect,
+signed ID Tokens can be stored by Relying Parties. In the case of
+SD-JWT, Holders have to store signed SD-JWTs and associated Disclosures,
+and Issuers and Verifiers may decide to do so as well.
+
+As usual, a leak of such data risks revealing private data of End-Users
+to third parties. As discussed in (#holder_binding_security), leaked
+SD-JWTs may also allow attackers to impersonate Holders unless Holder
+Binding is enforced and the attacker does not have access to the
+Holder's cryptographic keys. For these reasons, and in particular
+because the authenticity of the authority-signed data is easy to verify,
+the data may have a high monetary value on black markets.
```

Comment:
Is this what is meant? I was struggling with `because the authenticity of the authority-signed data is easy to verify,`

```suggestion
to third parties. As discussed in (#holder_binding_security),
because the authenticity of the authority-signed data is easy to verify,
leaked SD-JWTs may also allow attackers to impersonate Holders, unless Holder
Binding is enforced and the attacker does not have access to the
Holder's cryptographic keys. For these reasons,
the data may have a high monetary value on black markets.
```

Reply:
What I want to say is that the signature increases the value of the data (because anyone can verify that the data is authentic).

Reply:
```suggestion
to third parties. Signed End-User data, the authenticity of which
can be easily verified by third parties, further exacerbates the risk.
As discussed in (#holder_binding_security), leaked
SD-JWTs may also allow attackers to impersonate Holders unless Holder
Binding is enforced and the attacker does not have access to the
Holder's cryptographic keys. Altogether, leaked SD-JWT credentials may have
a high monetary value on black markets.
```